docs/releases.md

```diff
@@ -1,39 +1,157 @@
+
+### Release Stage 2 - Invite IPFS partners to test
+
+- [ ] Reach out to the following IPFS partners for testing this release (check when no more problems have been reported):
+  - [ ] Infura
+  - [ ] Textila
+  - [ ] Open Bazaar
+  - [ ] QRI
+- [ ] Run tests available in the following repos with the latest RC (check when all tests pass):
+  - [ ] [orbit-db](https://github.com/orbitdb/orbit-db)
+
+PSA: If you are a heavy user of `go-ipfs`, have developed a solid test infrastructure for your application and would love to help us would like to help us test `go-ipfs` release candidates, reach out to go-ipfs-wg@ipfs.io.
+
+### Release Stage 3 - Announce to the broader community
+
+- [ ] Documentation
+  - [ ] Ensure that CHANGELOG.md is up to date
+  - [ ] Ensure that README.md is up to date
+  - [ ] Ensure that all the examples we have produced for go-ipfs run without problems
+  - [ ] Update HTTP-API Documentation on the Website using https://github.com/ipfs/http-api-docs
+- [ ] Invite the community through (link to the release issue):
+  - [ ] [discuss.ipfs.io](https://discuss.ipfs.io/c/announcements)
+  - [ ] Twitter
+
+### Release Stage 4 - Complete the Release
+
+- Take a snapshot between of everyone that has contributed to this release (including its subdeps in IPFS, libp2p, IPLD and multiformats) using [`name-your-contributors`](https://www.npmjs.com/package/name-your-contributors). Generate a nice markdown list with [this script](https://gist.github.com/alanshaw/5a2d9465c5a05b201d949551bdb1fcc3). Add it to this issue.
+
+- [ ] Final preparation
+  - [ ] Verify that version string in `repo/version.go` has been updated
+  - [ ] tag commit with vX.Y.Z
+  - [ ] update release branch to point to release commit (`git merge vX.Y.Z`).
+  - [ ] publish dist.ipfs.io
+  - [ ] publish next version to https://github.com/ipfs/npm-go-ipfs
+- [ ] Publish a Release Blog post (at minimum, a c&p of this release issue with all the highlights, API changes, link to changelog and thank yous)
+- [ ] Broadcasting (link to blog post)
+  - [ ] Twitter
+  - [ ] IRC
+  - [ ] Reddit
+  - [ ] [discuss.ipfs.io](https://discuss.ipfs.io/c/announcements)
+  - [ ] Announce it on the [IPFS Users mlist](https://groups.google.com/forum/#!forum/ipfs-users)
+
+# ❤️ Huge thank you to everyone that made this release possible
+
+In alphabetical order, here are all the humans that contributed to the release:
+
+- <use script -- listed in Release Stage 4 -- to generate a list of everyone that contributed for this release>
+
+# 🙌🏽 Want to contribute?
+
+Would you like to contribute to the IPFS project and don't know how? Well, there are a few places you can get started:
+
+- Check the issues with the `help wanted` label in the [go-ipfs repo](https://github.com/ipfs/go-ipfs/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22)
+- Join an IPFS All Hands, introduce yourself and let us know where you would like to contribute - https://github.com/ipfs/team-mgmt/#weekly-ipfs-all-hands
```

Comment:
Maybe we should start pointing people at IRC/Matrix instead of the All Hands.

Reply:
Multiple people have reported that showing up on the main channel on IRC with ideas and questions typically results into them falling into oblivion.

If someone is new, it is nice to see that there are humans, that is why the call is important.

Reply:
Fair enough.